src/oc_bifrost/Makefile

```diff
@@ -16,6 +16,13 @@ endif
 
 # If the user hasn't set POSTGRES_USER, default to postgres
 POSTGRES_USER ?= 'postgres'
+POSTGRES_CMD = sudo -u  postgres psql -U $(POSTGRES_USER)
```

Comment:
Let's revert all of the changes here, except for the travis: ci lines below.

In other words, let's make the only change to this file parallel to the changes you made in the other makefiles.